.github/ISSUE_TEMPLATE/90_tech-checklist.md

```diff
@@ -0,0 +1,90 @@
+---
+name: "[Tech Chair] Infrastructure Setup Checklist"
+about: Checklist for Tech chair to fill out to set up repository
+title: "[PRE-REVIEW] Infrastructure Setup Checklist"
+labels: pre-review
+assignees: ''
+---
+
+<!--
+## DO NOT EDIT THIS FILE OUTSIDE OF THE journalovi/jovi-workflows REPOSITORY
+##
+## This file is automatically updated in all repositories within the journalovi
+## Github organization whenever the version in journalovi/jovi-workflows is
+## changed, so any other edits will be overwritten. To update this file, make
+## a commit or pull request at https://github.com/journalovi/jovi-workflows
+-->
+
+- [ ] Set up Github pages
+   - [ ] Under **Settings -> Actions -> General**:
+      - [ ] Check "Allow journalovi, and select non-journalovi, actions".
+      - [ ] Under "Allow specified actions and reusable workflows", enter:
+         ```
+         actions/*,quarto-dev/*
+         ```
+         !image
+   - [ ] If the paper is static HTML and does not require compilation:
+      - [ ] ensure the paper contents are in `index.html` in the root folder of the repository
+      - [ ] Go to **Settings -> Pages**:
+         - [ ] Set "Source" to "Deploy from a branch"
+         - [ ] Set "Branch" to "main" (or "master")
+         - [ ] Set the folder to "/"
+   - [ ] If the paper requires compilation:
+      - [ ] Ensure it has a workflow file, such as `.github/workflows/publish.yml`, that compiles the paper and copies the compiled result to the root folder of the `gh-pages` branch. Some example workflow files:
+         - [Quarto](https://github.com/journalovi/jovi-template-quarto/blob/main/.github/workflows/publish.yml)
+         - [Idyll](https://github.com/journalovi/2024-Cashman-PAC-learning-game/blob/master/.github/workflows/publish.yml) - this template can be easily adapted for other workflows that use custom tools for building, e.g. npm or yarn, by swapping out the `npm` commands and ensuring the correct output folder is copied to the `gh-pages` branch in the last step.
+      - [ ] Go to **Settings -> Pages**:
+         - [ ] Set "Source" to "Deploy from a branch"
+         - [ ] Set "Branch" to "gh-pages"
+         - [ ] Set the folder to "/"
+      - [ ] Verify the compilation works. On the repo page, go to Actions and click on the "XXX Publish" (XXX = Quarto/Idyll/etc) workflow on the left, then click Run Workflow and run it.
+   - [ ] Verify that the "pages build and deployment" action is successful.
+
+   - [ ] Check that the live version of the article is visible at http://www.journalovi.org/20XX-YY-ZZ (replace the last part with the repo name).
+
+
+- [ ] Ensure the paper contains a notice infobox at the top that the paper is under review and that the repository links in the notice are correct.
```

Comment:
Add after "are correct": "by replacing the `20xx-repo-name` part with the repository URL."

Reply:
done